src/index.html

```diff
@@ -11,6 +11,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://www.google.com/"
+      method="get"
+      data-qa="big"
+      class="bigField field"
```

Comment:
```suggestion
      class=" field"
```
![Screenshot from 2023-03-10 13-24-13](https://user-images.githubusercontent.com/33029864/224304135-dd8f51c7-411d-4c26-8520-4dcd2da23d27.png)
![Screenshot from 2023-03-10 13-23-18](https://user-images.githubusercontent.com/33029864/224304140-89353227-c6cf-4cf6-ba39-5a4394e18528.png)